clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```
@@ -1543,11 +1855,181 @@
     "body": ["NAMESPACE ${1:item}"],
     "description": "NAMESPACE-prefix ITEM"
   },
+  "NATIONAL-EDITED #1": {
+    "prefix": "NATIONAL-EDITED",
+    "body": [
+      "WORKING-STORAGE SECTION.",
+      "01 ${2:ORIG-CUSTOMER-NAME} GROUP-USAGE ${1|NATIONAL,NATIONAL-EDITED|}",
```

```suggestion
      "01 ${2:ORIG-CUSTOMER-NAME} GROUP-USAGE NATIONAL.",
```

```
  000009                01 ORIG-CUSTOMER-NAME GROUP-USAGE NATIONAL-EDITED.                     
                                                                                                
==000009==> IGYDS0093-S "NATIONAL-EDITED" was found in the "GROUP-USAGE" clause.  The clause was
                        discarded.                                                              
```

The snippet name and prefix will need to be adjusted as well.